elementary/config/config.py

```diff
@@ -69,10 +72,12 @@ def __init__(
 
         config = self._load_configuration()
 
-        self.target_dir = self._first_not_none(
+        self.target_dir = self.target_dir = self._first_not_none(
```

Comment:
Typo without side effects but still - let's fix: 

```suggestion
        self.target_dir = self._first_not_none(
```